packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -485,7 +488,7 @@ The Domain object defines the following properties:
 
 | Property                | Type          | Description |
 | ----------------------- | -------------------- | ---------------------------------------------------------------------------------------------------------- |
-| `certificateSourcetype` | `MANUAL`             | Required. Certificate source type that indicates whether the Certificate is provided by the user. Accepted value: `MANUAL`|
+| `certificateSourcetype` | String            | Required. Certificate source type that indicates whether the Certificate is provided by the user or Okta.  Accepted values: `MANUAL`, `OKTA_MANAGED`. **Warning:** Use of `OKTA_MANAGED` Requires a feature flag to be enabled.|
```

Comment:
Requires the EA feature "Custom Domains with Okta-Managed Certificates" to be enabled.

Reply:
@brentschaus-okta is this ok?